programs/ssl/ssl_server2.c

```diff
@@ -1996,12 +1998,23 @@ int main( int argc, char *argv[] )
             if( ret != 0 )
                 break;
         }
+        if( ret == 0 )
```

Comment:
Why do we want to parse der certificates only if pem ones were succesfull?

Reply:
Because a parsing error should be treated as fatal here.